Backend-tests.md

```diff
@@ -151,33 +151,57 @@ In fact, too many mocks can be a problem because when someone changes the code y
 
 You can run backend tests like this:
 
+Python:
 ```console
 python -m scripts.run_backend_tests
 ```
 
+Docker:
+```console
+make run_tests.backend
+```
+
 Alternatively, you can run just a single test module like this:
 
+Python:
 ```console
 python -m scripts.run_backend_tests --test_target=core.controllers.editor_test
 ```
 
+Docker:
+```console
+make run_tests.backend PYTHON_ARGS="--test_target=core.controllers.editor_test"
+```
+
 The argument to `--test_target` can be as specific as you like. For example:
 
 * Run a class of tests: `--test_target=core.controllers.editor_test.BaseEditorControllerTests`
 * Run a single test: `--test_target=core.controllers.editor_test.BaseEditorControllerTests.test_editor_page`
 
 If you also want to see the output of `print` statements and error logs in the terminal, use `--verbose` like this:
 
+Python:
 ```console
 python -m scripts.run_backend_tests --test_target=core.controllers.editor_test --verbose
 ```
 
+Docker:
+```console
+make run_tests.backend PYTHON_ARGS="--test_target=core.controllers.editor_test --verbose"
```

Comment:
this is also thworing same 
![Screenshot from 2024-02-08 18-11-21](https://github.com/oppia/oppia-web-developer-docs/assets/127002519/00795d1c-7808-4a1a-a27e-26f6bdd75fbf)